backend/src/packages/orders/libs/exceptions/order-not-found.ts

```diff
@@ -0,0 +1,12 @@
+import { HttpMessage } from '~/libs/enums/enums.js';
+import { NotFoundError } from '~/libs/exceptions/exceptions.js';
+
+class OrderNotFound extends NotFoundError {
+  public constructor() {
+    super({
+      message: HttpMessage.ORDER_DOES_NOT_EXIST,
+    });
+  }
+}
```

Comment:
again, can't you reuse not found error and throw there needed message? now it looks like same error but with the other message